roles/restore/tasks/deploy_awx.yml

```diff
@@ -8,9 +8,30 @@
       bash -c "cat '{{ backup_dir }}/awx_object'"
   register: awx_object
 
-- name: Set AWX spec variable from backup
+- name: Create temp file for spec dict
+  tempfile:
+    state: file
+  register: tmp_spec
```

Comment:
If we could use `| to_json` when writing the spec to the backup, this business of creating a temp file, writing to it, then including vars from it could be avoided.  When [writing the file](https://github.com/ansible/awx-operator/pull/133/files#diff-3ced427495f8e83eeeefcf8bf0228eb0c2b7f0398758d64a46b49954d05528bcR24), we can't use `to_json` because the quotes in the resulting diction will be lost, so ansible treats it as an AnsibleUnsafeText type.  

If we could find a way to keep the quotes, we could just do:
```
- set_fact:
    awx_spec: "{{ lookup('file', 'awx_object') | from_json }}"
```

Reply:
The feedback loop is slow, so here is a quick reproducer: https://gist.github.com/rooftopcellist/42a48b6d59488182176f8f5faa3fef0e

And here is my fixed version of that reproducer (which is used in the PR currently): https://gist.github.com/rooftopcellist/2943599dd731d5bfe4d160b93d37274e